hydra-tx/testlib/Test/Hydra/Tx/Gen.hs

```diff
@@ -114,19 +128,11 @@ shrinkUTxO = shrinkMapBy (UTxO . fromList) UTxO.pairs (shrinkList shrinkOne)
       | value' <- shrinkValue value
       ]
 
--- | Generate a complete arbitrary UTxO, which may contain arbitrary assets in
--- 'TxOut's addressed to public keys *and* scripts. NOTE: This is not reducing
--- size when generating assets in 'TxOut's, so will end up regularly with 300+
--- assets with generator size 30. NOTE: The Arbitrary TxIn instance from the
--- ledger is producing colliding values, so we replace them.
+-- | Generate a 'Conway' era 'UTxO'. See also 'genTxOut'.
 genUTxO :: Gen UTxO
-genUTxO = do
-  utxoMap <- Map.toList . Ledger.unUTxO <$> arbitrary
-  fmap UTxO.fromPairs . forM utxoMap $ \(_, o) -> do
-    i <- arbitrary
-    pure (i, fromLedgerTxOut o)
+genUTxO = sized genUTxOSized
```

Comment:
minor: shouldn't genUTxOSized be actually sized?
i think we should swap names here

Reply:
Both are sized :) `genUTxO` is sized implicitly through quickcheck, where `genUTxOSized` is sized explicitly.